### ISSUE TITLE
Do we want to re-order the activity history page when a workorder attempt is re-run?

### ISSUE THREAD

**Issue by taylordowns2000** [OpenFn/Lightning#1360]
Right now we don't. And that might be OK. But we also might want it jumping up to the top? Smoothly? Lol. @NickOpenFn , what should we do here?

**Comment by NickOpenFn** [comment]
@taylordowns2000 I don't think we should even touch this now and should hit this with a holistic improvement to the History page as a single epic.
